views/layouts/main.handlebars

```diff
@@ -51,7 +51,7 @@
                             <li><a class="inner-link {{#compare menuActive '==' 'home'}}active{{/compare}}" href="/" title="Home">Home</a></li>
                             <li><a class="inner-link {{#compare menuActive '==' 'churchService'}}active{{/compare}}" href="/church-service" title="Church Service">Church Service</a></li>
                             <li><a class="inner-link {{#compare menuActive '==' 'kingsPay'}}active{{/compare}}" href="/kings-pay" title="Kings Pay">Kings Pay</a></li>
-                            <li><a class="inner-link  {{#compare menuActive '==' 'pcdlOnline'}}active{{/compare}}" href="/pcdl-online" title="PCDL Online">PCDL Online</a></li>
+                            <li><a class="inner-link  {{#compare menuActive '==' 'pcdlOnline'}}active{{/compare}}" href="https://pcdl.co/" title="PCDL Online">PCDL Online</a></li>
```

Comment:
why are we linking the pcdl site outside our project?
we should have a local page for it like kingschat, which will have a brief description, and then lead users to the main pcdl page